gensrc/thrift/Descriptors.thrift

```diff
@@ -51,6 +51,8 @@ struct TSlotDescriptor {
   9: required i32 slotIdx
   10: required bool isMaterialized
   11: optional i32 col_unique_id = -1
+  12: optional bool is_key = false
+  13: optional bool is_invalid = false
```

Comment:
add some comments for `is_invalid`. To be honest, I think this is a bit confusing. How about to rename it to `should_ignore` or `need_materialize`?

Reply:
good advice

Reply:
good advice